soccer/gameplay/plays/Attack_Goal/distraction_attack.py

```diff
@@ -230,11 +230,12 @@ def on_enter_passing(self):
     def on_enter_cross(self):
         self.remove_all_subbehaviors()
         #if the ball is passed to the distractor the ball is passed to the striker, as the third robot moves to the right to distract more
-        #add chip here if pass chance is low
-        self.add_subbehavior(
-            tactics.coordinated_pass.CoordinatedPass(self.striker_point),
-            'pass to striker',
-            required=True)
+        ball = main.ball().pos
+        self.striker_point.x = -(ball.x / 2 + constants.Field.Width / 4)
```

Comment:
Do you always want this to happen on the right side of the field? You may want to reflect the play depending on robot positions.

Reply:
I would suggest some randomness here, but at the very least pick which side to go on in an intelligent fashion. Although, we should really integrate pass optimization here in the long run.